sdk/automanage/Microsoft.Azure.Management.AutoManage/src/Microsoft.Azure.Management.AutoManage.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
+  <!-- Please do not move/edit code below this line -->
+  
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
I think the errors are related to the TargetFramework being explicitly coded here. I also notice that you are missing related packageid tags etc. Please reference existing .csproj for other RPs.

Reply:
thank you very much, done.